src/op_mode/vrf.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2022 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import sys
+
+from tabulate import tabulate
+from vyos.util import cmd
+
+import vyos.opmode
+
+
+def get_raw_data():
+    """
+    :return: list
+    """
+    command = cmd('sudo ip --json --brief link show type vrf')
+    data = json.loads(command)
+    return data
+
+
+def get_vrf_members(vrf: str) -> list:
+    """
+    Get list of interface VRF members
+    :param vrf: str
+    :return: list
+    """
+    command = cmd(f'sudo ip --json --brief link show master {vrf}')
+    answer = json.loads(command)
+    interfaces = []
+    for data in answer:
+        if 'ifname' in data:
+            interfaces.append(data.get('ifname'))
+    return interfaces if len(interfaces) > 0 else ['n/a']
+
+
+def get_formatted_output(raw_data):
```

Comment:
Oh, also, we may want to use `_get_formatted_output` etc. to mark these functions private. Not critical, but I think it will make their role clearer.

Reply:
Done!